torch_glow/src/GlowFuser.cpp

```diff
@@ -127,20 +127,28 @@ bool canMerge(torch::jit::Block *block, isSupportFunc fn) {
 
 #define REQ(cond, log_info)                                                    \
   if (!(cond)) {                                                               \
-    llvm::errs() << log_info;                                                  \
+    llvm::errs() << log_info << "\n";                                          \
     return c10::nullopt;                                                       \
   }
 
+// Check if a node is a known not-lowered operator, etc, a tensor generator.
+bool isLogIgnoredOp(const std::string &op_name) {
```

Comment:
we do not use '_' in names, opName.

Same in other places.

Reply:
oops, I used this convention for all the naming in torch_glow...

Reply:
We shouldn't use snake names, only camel case. The exception in torch_glow should be the stuff like the code in FuseLinear where it is copied basically straight from torch/tvm and I'm fine with no adapting it to glow because I would like to move it into a common place in pytorch. Other than that, we should be sticking with glow's regular style norms